plugins/techdocs-backend/src/search/DefaultTechDocsCollator.ts

```diff
@@ -30,34 +31,57 @@ interface MkSearchIndexDoc {
   location: string;
 }
 
+type TechDocsCollatorOptions = {
```

Comment:
this should probably actually be exported all the way out to the API so it appears in the api-report and makes the `ae-forgotten-export` go away. We know there's lots of those but we're trying to whittle them away. :)